server/src/transport/ws.rs

```diff
@@ -272,11 +272,11 @@ pub(crate) async fn background_task<L: Logger>(
 				stopped = stop;
 				permit
 			}
-			None => break Ok(()),
+			None => break Ok(Shutdown::ConnectionClosed),
```

Comment:
Am I right in thinking that:
- `Stopped` means the user has manually stopped the server, so connection was closed
- `ConnectionClosed` means the connection was closed for some other reason (eg network issue or whatever)

Reply:
yes, this is only fails if the `send_task` has been completed and the receiver of the bounded channel has been dropped.

probably some other variant is in appropriate  here then